app/objects/DraftModel.js

```diff
@@ -4,5 +4,11 @@ export default class DraftModel {
 		this.name = name;
 		this.games = games;
 		this.players = players;
+        this.Values = {
```

Comment:
lower case?

Reply:
I added it to support the structure that exists on resources/js/custom/app.js
when i pass the object to App.Data it was replacing this part and causing errors. There are a few references to refactor to make it lower. Can do it in the next iteration